doc/acl.md

```diff
@@ -1 +1,762 @@
+    }
+}
+```
+
+## <a name="view"></a>View
+
+### Endpoint
+GET /access/:id
+
+### Headers
+Key | Value 
+--- | ---
+Content-Type | application/json
+Accept | application/json
+Email | cdc_user@gmail.com
+
+### Response Payloads
+HTTP Code | Status | Description
+--- | --- | ---
+404 | Not Found | Access not found in database  
+500 | Internal Server Error | some un-handle error in server 
+200 | OK | OK
+
+{
+    "status_code": "CDC-200",
+    "status_message": "OK",
+    "data": {
+        "access": {
+                "id":2,
+                "parent_id": 1,
+                "alias": "groups",
+                "childrens": [
+                        {
+                            "id": 3,
+                            "parent_id": 2,
+                            "alias": "groups:index",
+                            "children": null
+                        },
+                        {
+                            "id": 4,
+                            "parent_id": 2,
+                            "alias": "groups:view",
+                            "children": null
+                        }
+                ]
+            }
+            
+        }
+    
+}
+
+### Scenario Test
+
+#### Case : Positive Case 1
+
+Response HTTP Status Code : 200
+
+Response Payload :
+```
+{
+    "status_code": "CDC-200",
+    "status_message": "OK",
+    "data": {
+        "access": {
+            "id": 1,
+            "parent_id": null,
+            "alias": "controllers",
+            "childrens": null
+        }
+    }
+}
+```
+
+#### Case : Positive Case 2
+
+Response HTTP Status Code : 200
+
+Response Payload :
+```
+{
+    "status_code": "CDC-200",
+    "status_message": "OK",
+    "data": {
+        "access": {
+            "id": 1,
+            "parent_id": null,
+            "alias": "controllers",
+            "childrens": [
+                {
+                    "id":2,
+                    "parent_id": 1,
+                    "alias": "groups",
+                    "childrens": [
+                        {
+                            "id": 3,
+                            "parent_id": 2,
+                            "alias": "groups:index",
+                            "children": null
+                        }
+                    ]
+                },
+                {
+                    "id": 6,
+                    "parent_id": 1,
+                    "alias": "users",
+                    "childrens": [
+                        {
+                            "id": 7,
+                            "parent_id": 6,
+                            "alias": "users:create",
+                            "childrens": null
+                        },
+                        
+                    ]
+                }
+            ]
+        }
+    }
+}
+```
+
+
+## <a name="edit"></a>Edit
+
+### Endpoint
+PUT /access/:id
+
+### Headers
+Key | Value 
+--- | ---
+Content-Type | application/json
+Accept | application/json
+Email | cdc_user@gmail.com
```

Comment:
also at PUT /access/:id, it's doesn't email header. Because there is no filtering data in the endpoint. so remove email on header. thanks.